lib/Unexpected.js

```diff
@@ -940,6 +951,9 @@ Unexpected.prototype.lookupAssertionRule = function (subject, testDescriptionStr
         if (!matches(subject, handler.subject.type, 'subject', relaxed)) {
             return false;
         }
+        if (requireAssertionSuffix && !handler.args.some(function (arg) { return arg.type.is('assertion'); })) {
```

Comment:
Let's use Gustav style and break the condition into multiple lines or make some variables.